content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -0,0 +1,136 @@
+# Billing and Payment
+
+Studio is free for individuals and small teams (up to 2 members). The Free plan
```

Comment:
Instead of "small teams", how about "teams with 2 members."

Reply:
or "two members"

Reply:
Changed.